core/sector_storage/stores/impl/local_store.cpp

```diff
@@ -42,13 +50,21 @@ namespace fc::sector_storage::stores {
     return StoreErrors::kInvalidSectorName;
   }
 
-  LocalStoreImpl::LocalStoreImpl(std::shared_ptr<LocalStorage> storage,
-                                 std::shared_ptr<SectorIndex> index,
-                                 gsl::span<const std::string> urls)
+  LocalStoreImpl::LocalStoreImpl(
+      std::shared_ptr<LocalStorage> storage,
+      std::shared_ptr<SectorIndex> index,
+      gsl::span<const std::string> urls,
+      std::shared_ptr<boost::asio::io_context> context,
+      Ticks ticks)
       : storage_(std::move(storage)),
         index_(std::move(index)),
-        urls_(urls.begin(), urls.end()) {
+        urls_(urls.begin(), urls.end()),
+        context_(std::move(context)) {
     logger_ = common::createLogger("Local Store");
+    std::shared_ptr<host::HostContext> fsm_context =
+        std::make_shared<host::HostContextImpl>(context_);
+    scheduler_ = std::make_shared<libp2p::protocol::AsioScheduler>(
+        *fsm_context->getIoContext(), libp2p::protocol::SchedulerConfig{ticks});
```

Comment:
`HostContext` is unnecessary, because `getIoContext()` returns same as `context_` and `HostContext` is not used otherwise